src/icons/states/RelayLiteEmptyState.js

```diff
@@ -0,0 +1,352 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg/AccessibleSVG';
+import { bdlBoxBlue, bdlBoxBlue20 } from '../../styles/variables';
+
+import type { Icon } from '../flowTypes';
+
+const RelayLiteEmptyState = ({ className = '', height = 140, title, width = 140 }: Icon) => (
```

Comment:
Can this file be cleaned up/re exported? Lots of transforms and g tags that are likely unnecessary